2019-05-23 12:28:27.046908: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2019-05-23 12:28:27.203771: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1432] Found device 0 with properties: 
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.582
pciBusID: 0000:03:00.0
totalMemory: 10.92GiB freeMemory: 10.76GiB
2019-05-23 12:28:27.203803: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1511] Adding visible gpu devices: 0
2019-05-23 12:28:34.092612: I tensorflow/core/common_runtime/gpu/gpu_device.cc:982] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-05-23 12:28:34.092665: I tensorflow/core/common_runtime/gpu/gpu_device.cc:988]      0 
2019-05-23 12:28:34.092681: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1001] 0:   N 
2019-05-23 12:28:34.093228: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10409 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:03:00.0, compute capability: 6.1)
Using TensorFlow backend.
Traceback (most recent call last):
  File "MANN_vary_numberofkeys.py", line 197, in <module>
    results = fit_evaluate(model1, x_train_, y_train_, x_test, y_test, batch_size, epochs, lr)
  File "MANN_vary_numberofkeys.py", line 161, in fit_evaluate
    memory = history.layers[-1].get_memory()
AttributeError: 'History' object has no attribute 'layers'
srun: error: gpu-010: task 0: Exited with exit code 1
